pkg/configs/client/client.go

```diff
@@ -0,0 +1,151 @@
+package client
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/cortexproject/cortex/pkg/configs"
+	"github.com/cortexproject/cortex/pkg/configs/db"
+	"github.com/cortexproject/cortex/pkg/util"
+	"github.com/go-kit/kit/log/level"
+)
+
+// Client is what the ruler and altermanger needs from a config store to process rules.
+type Client interface {
```

Comment:
This interface looks to be missing a `GetTemplates` method for alert template files, which were added in PR #1237.

Reply:
Hi @khaines - I don't understand - I don't see a `GetTemplates` method in that PR (or anywhere in the codebase).  What am I missing?

Reply:
Yeah, I think we're good - GetAlerts returns a ConfigsResponse which contains the templates, like before.